.github/workflows/lint.yml

```diff
@@ -48,6 +48,22 @@ jobs:
         with:
           src: ${{ env.BLACK_SRC }}
           options: "--line-length ${{ env.BLACK_LINE_LENGTH }} --extend-exclude '${{ env.BLACK_EXTEND_EXCLUDE }}' --check --verbose"
+  pylint-for-python:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'pull_request' || github.event.pull_request.draft == false
+    env: 
+      PYLINT_LINE_LENGTH: 92
+      PYLINT_SRC: 'flepimop/gempyor_pkg/src/gempyor'
```

Comment:
Can we configure the line length with a `.pylintrc` file instead of manually specifying via command line? I think we'll need an rc file later anyways if we want to tune the rules for our uses.